tests/unittest.py

```diff
@@ -14,21 +14,30 @@
 # limitations under the License.
 import twisted
 from twisted.trial import unittest
+from synapse.util.logcontext import LoggingContextFilter
+from twisted.logger import Logger, LogLevel
```

Comment:
Maybe we should use absolute names for these (`import twisted.logger; tx_log=twisted.logger.Logger()`) to reduce confusion between twisted and stdlib logging infrastructure